src/modules/IsnsServer.rb

```diff
@@ -293,16 +301,32 @@ def Read
     end
 
     # Write all iscsi-server settings
-    # @return true on success
+    #
+    # @return [Boolean] true on success; false otherwise
     def Write
       # IsnsServer write dialog caption
       caption = _("Saving isns Configuration")
 
       Y2Firewall::Firewalld.instance.write
 
-      # set isns initscript status
-      return false if !setServiceStatus
-      true
+      if Mode.auto || Mode.commandline
+        # set isns initscript status
```

Comment:
np: I would update the comment (it refers to the old System V init scripts which have been replaced by Systemd).